### ISSUE TITLE
installation problem

### ISSUE THREAD

**Issue by arademaker** [letuananh/lelesk#1]
The install procedure is not clear. It looks like lelesk is installed with NLTK? I installed nltk and after that I got

```
$ python3 lelesk/wsdtk.py -W car -x 'the car is on the road'
Traceback (most recent call last):
  File "lelesk/wsdtk.py", line 60, in <module>
    from .main import LeLeskWSD, LeskCache
ModuleNotFoundError: No module named '__main__.main'; '__main__' is not a package
leme:lelesk ar$ pwd
/Users/ar/work/lelesk
```

Not clear how to install this package. Can you elaborate the instructions?

**Comment by letuananh** [comment]
I'm packing this package to release to PyPI soon. Currently I'm working on the setup script and uploading data. After that it can be installed via PyPI. Thank you for contributing.

**Comment by letuananh** [comment]
This has been fixed with lelesk version 0.1 release: https://github.com/letuananh/lelesk/releases/tag/lelesk-0.1

LeLesk is now available on [PyPI](https://pypi.org/project/lelesk/) and can be installed with pip

```bash
pip install lelesk
```

Fore more information please see https://pypi.org/project/lelesk/ Thank you :)
